src/PublicApiGeneratorTests/Field_modifiers.cs

```diff
@@ -42,14 +42,31 @@ public void Include_const_fields()
             // Have to include the ctor - I can't figure out how to hide it
             // when values are initialized
             AssertPublicApi<ClassWithConstFields>(
-@"namespace PublicApiGeneratorTests.Examples
+                @"namespace PublicApiGeneratorTests.Examples
 {
     public class ClassWithConstFields
     {
         protected const string ConstProtectedField = ""hello world"";
         public const int ConstPublicField = 42;
         public ClassWithConstFields() { }
     }
+}");
+        }
+
+        [Fact]
+        public void Include_unsafe_fields()
+        {
+            // Have to include the ctor - I can't figure out how to hide it
+            // when values are initialized
+            AssertPublicApi<ClassWithUnsafeFields>(
+                @"namespace PublicApiGeneratorTests.Examples
+{
+    public class ClassWithUnsafeFields
+    {
+        protected unsafe System.Void* UnsafeProtectedField;
```

Comment:
`System.Void` should be changed later to `void` by #115

Reply:
Btw I'm happy to wait till your PR is merged and rebase mine so that these PRs aren't a burden.

Reply:
#115 has grown much wider than I expected

Reply:
I rebased this branch on master after #115 was merged. Apparently it did not change to `void`.

Reply:
I was wrong. See jnm2/ApiApprover#1